bicycleparameters/main.py

```diff
@@ -8,7 +8,7 @@
 import matplotlib.pyplot as plt
 from matplotlib.patches import Ellipse, Wedge
 from uncertainties import unumpy
-from dtk import control
+#from dtk import control
```

Comment:
If this is causing issues for you, you need to make sure dynamicisttoolkit is installed. Undo this comment.

Reply:
I searched online for a dynamicstoolkit or dtk module but I wasn't able to find anything. It never gave me issues before, but my python versions have gotten a little messed up and that likely had something to do with it.

Import has been uncommented.

Reply:
How are you installing packages? pip or conda or other?

Reply:
You should be able to do `pip install DynamicistToolkit` then.

Reply:
Done. Big help.